README.md

```diff
@@ -1,60 +1,56 @@
-# Awesome-books-project
-
-AwesomeBooks
-A basic website that allows users to add/remove books from a list.
-
-Built With
-Major languages: HTML, CSS & Javascript
-Technologies used: Linters, Git and Github
-Live Demo
-https:
-
-Getting Started
-Prerequisites
-You should be having Node.js installed on your computer
-
-Setup
-To get a local copy up and running follow these simple example steps.
-
-Navigate to a folder where you want the cloned file to appear
-
-Clone this repo in your terminal or git bash using the command
-
-git@github.com:KennyEgun/AwesomeBooks.git
-run npm install to install all the necessary dependencies including the linters
-
-Navigate to the folder called capstone-one and open this project using your editor
-
-Run tests
-These were linter tests mainly
-
-For css, run npx hint .
-Html, run npx stylelint "\*_/_.{css,scss}"
-Javascript npx eslint .
-Deployment
-
-## Author 1
-
-👤 **JAMES ASIBEY**
-
-- GitHub:[https://github.com/jaamanu](https://github.com/jaamanu)
-- Twitter:[@JamesAsibeyManu](https://twitter.com/JamesAsibeyManu)
-- LinkedIn:(https://www.linkedin.com/in/jamesasibeymanu)
-
-
-
-🤝 Contributing
-Contributions, issues, and feature requests are welcome!
-
-Feel free to check the issues page.
-
-Show your support
-Give a ⭐️ if you like this project!
-
-Acknowledgments
-Hat tip to anyone whose code was used
-Inspiration
-etc
-📝 License
-This project is MIT licensed.
-# AWESOME-BOOKLIST-ES6
+# Awesome-books-project
+
+AwesomeBooks
+A refactoring basic web application that allows users to add/remove books from a list.
+
+Built With
+Major languages: HTML, CSS & Javascript
+Technologies used: Linters, Git and Github
+Live Demo(https://jaamanu.github.io/AWESOME-BOOKLIST-ES6/)
+https:[https://jaamanu.github.io/AWESOME-BOOKLIST-ES6/]
+
+Getting Started
+Prerequisites
+You should be have Javascript and Node.js installed on your computer
+
+Setup
+To get a local copy up and running follow these simple example steps.
+
+Navigate to a folder where you want the cloned file to appear
+
+Clone this repo in your terminal or git bash using the command
+
+https://github.com/jaamanu/AWESOME-BOOKLIST-ES6
+run npm install to install all the necessary dependencies including the linters
+
+Navigate to the folder called AWESOME-BOOKLIST-ES6 and open this project using your editor
+
+Prerequisites
+Setup
+
+Run tests
+These were linter tests mainly
+For css, run npx hint .
+Html, run npx stylelint "\*_/_.{css,scss}"
+Javascript npx eslint .
+Deployment
+
+👤 **JAMES ASIBEY**
+
+-GitHub:https://github.com/jaamanu
+-Twitter:@JamesAsibeyManu
+-LinkedIn:(https://www.linkedin.com/in/jamesasibeymanu)
+
+Feel free to check the issues page.
+
+Show your support
+Give a ⭐️ if you like this project!
+
+Acknowledgments
+Hat tip to anyone whose code was used
+Inspiration
+etc
+📝 License
+This project is MIT licensed.
```

Comment:
- [ ]  As requested by the previous reviewer, please make your README file more descriptive and professional way. As he said that there are no headings for each section. please see this [Template](https://github.com/microverseinc/readme-template) and use it